### DISCUSSION TITLE
可以部署在heroku上，建议新增一键部署功能

### DISCUSSION THREAD

**Discussion by eiyooooo** [spencerwooo/onedrive-vercel-index#35] [Ideas]
可以部署在heroku上
先fork项目，更改config文件夹里的两个文件
heroku上新建项目，连接到自己的项目
再deploy
最后再新建三个变量就能用了

建议作者根据[https://devcenter.heroku.com/articles/heroku-button](url)做一个一键部署

**Comment by spencerwooo** [comment]
可能暂时不会考虑 Heroku，毕竟项目叫 onedrive-**vercel**-index 不是白叫的。

**Reply by People-11** [reply]
确实，我测试过，Heroku在内地某些网络环境上无法良好的访问，应当优先考虑可访问性

**Reply by eiyooooo** [reply]
我这里vercel访问有问题，刷新几次才能打开，Heroku就没问题😂

**Reply by York618** [reply]
Vercel经常被墙，尤其是在西南地区会出现**dns解析问题**。
当然这不能怪作者，毕竟这玩意又不是**生产环境**使用。
**Star**只要一多起来，作者肯定出一个衍生版

**Reply by spencerwooo** [reply]
自备解决方案吧，全国各地的情况都不一样

**Reply by York618** [reply]
> 自备解决方案吧，全国各地的情况都不一样

一般我的方案是套cloudflare，但是并不太行，vercel不允许。再者，vercel自带aws hk的cdn
作者有时间还是出一个`onedrive-cloudflarepages-index`吧，毕竟cloudflare在中国pop点用的京东云不容易封

**Reply by aozaki-kuro** [reply]
https://vercel.com/support/articles/using-cloudflare-with-vercel#with-proxy
Vercel官方有给出与Cloudflare CDN联用的方案（当然官方不建议这么干）。
在页面规则里把`/.well-known/*`设置为“SSL 关”，以及关闭“强制HTTPS”即可通过Vercel的域名检查。
我这里的项目正常用上了。

另外Vercel自己也提供了特供版的CNAME `cname-china.vercel-dns.com`和特供IP `76.223.126.88`（未详细测试）

**Reply by spencerwooo** [reply]
> 作者有时间还是出一个 `onedrive-cloudflarepages-index` 吧

大佬请

**Reply by York618** [reply]
> > 作者有时间还是出一个 `onedrive-cloudflarepages-index` 吧
> 
> 大佬请

考虑下（🐶）

**Reply by York618** [reply]
> https://vercel.com/support/articles/using-cloudflare-with-vercel#with-proxy
> Vercel官方有给出与Cloudflare CDN联用的方案（当然官方不建议这么干）。
> 在页面规则里把`/.well-known/*`设置为“SSL 关”，以及关闭“强制HTTPS”即可通过Vercel的域名检查。
> 我这里的项目正常用上了。
> 
> 另外Vercel自己也提供了特供版的CNAME `cname-china.vercel-dns.com`和特供IP `76.223.126.88`（未详细测试）

感谢。最近这些天一直有些人说我站被墙了，这样应该好多了

**Reply by hmsjy2017** [reply]
@spencerwooo 
相比Heroku，部署在云函数可能体验更好，云函数是可选地域的。
